### ISSUE TITLE
Explain dephy-io.github.io  and  cross reference

### ISSUE THREAD

**Issue by stappersg** [dephy-io/website#1]
Both dephy-io/dephy-io.github.io  and dephy-io/website are about _"website"_.

Please explain what dephy-io.github.io is and if both need to exist, make cross references.

**Comment by jasl** [comment]
We were planning to use GitHub to host our website, but then we decided to use CF, so `dephy-io.github.io` is obsolete and will remove soon

**Comment by stappersg** [comment]
> so `dephy-io.github.io` is obsolete and will remove soon

Ah, okay.

(And that was quick I had this 
![image](https://github.com/dephy-io/website/assets/35434094/c5811a77-9e80-4b23-a88b-7acb98ac098a)
on my clipboard  :-)

**Comment by jasl** [comment]
I see. I just removed `dephy-io.github.io`.
For the content. We are still working hard, so we will keep polishing them when we making progress.

I appreciate your interest in us!

**Comment by stappersg** [comment]
On Sun, Oct 29, 2023 at 07:23:11AM -0700, Jun Jiang wrote:
> I appreciate your interest in us!

OK.


Groeten
Geert Stappers
Will be closing this issue.
-- 
Silence is hard to parse

**Comment by stappersg** [comment]
On Sun, Oct 29, 2023 at 07:15:29AM -0700, Jun Jiang wrote:
> Closed #1 as completed.

Oh, I missed that.


Groeten
Geert Stappers
-- 
Silence is hard to parse
